.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-env node */
 module.exports = {
-  extends: ['eslint:recommended', 'prettier', 'plugin:mdx/recommended', 'plugin:prettier/recommended'],
+  extends: ['eslint:recommended', 'prettier', 'plugin:mdx/recommended'],
```

Comment:
I think it's best practice to just have 'prettier': https://prettier.io/docs/en/integrating-with-linters.html